### DISCUSSION TITLE
Creating sub-users in one account

### DISCUSSION THREAD

**Discussion by rnenergy** [drakkan/sftpgo#1255] [Q&A]
Is it possible to create sub-users in one SFTPGO account through the web interface? Perhaps this option needs to be enabled in the configuration?

**Comment by drakkan** [comment]
What do you mean for sub-users? If user `a` has their home directory set to `/tmp/a` and you want user `b` to access `/tmp/a/b` just create a new user and set the home directory to `/tmp/a/b`. Obviously quota usage for user `a` will not be updated if you upload files as user `b`

**Comment by ErminMerdanovic** [comment]
Hi, I found myself pondering the same question as @rnenergy. I've already put in place the solutions you mentioned @drakkan. However, it appears as you described that the quota isn't being shared and updated between the two users. I thought that the sharing feature would be ideal for this situation, allowing the quota to be shared. Unfortunately, this feature is only available via the web and not through SFTP. Do you think there's a possibility of implementing SFTP to the share feature in the future? 

Perhaps it could involve using a username prefix like "a-b" for the shared account for connecting over SFTP. Alternatively, a solution could involve creating the user as a flagged subuser within the overall user system, where they also maintaining the same quota but with different sub folder access.

By the way, kudos for creating such an outstanding product!
